elixir/apps/domain/lib/domain/auth.ex

```diff
@@ -594,7 +552,9 @@ defmodule Domain.Auth do
         |> Authorizer.for_subject(Identity, subject)
         |> Repo.update_all(set: [deleted_at: DateTime.utc_now(), provider_state: %{}])
 
-      {:ok, _groups} = Actors.update_dynamic_group_memberships(assoc.account_id)
+      # TODO: IDP Sync
```

Comment:
I actually think this will be a no-op since we don't delete the actor if its last identity was deleted, but I left it in for now.